src/partition/src/partition.rs

```diff
@@ -72,8 +72,7 @@ impl Display for PartitionDef {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         write!(
             f,
-            "({}) VALUES LESS THAN ({})",
-            self.partition_columns.iter().join(", "),
+            "VALUES LESS THAN ({})",
```

Comment:
The `VALUE LESS THAN` is unnecessary too.
```suggestion
            "{}",
```